### ISSUE TITLE
Typescript Support

### ISSUE THREAD

**Issue by BengtHagemeister** [seatsio/seatsio-js#109]
Hi,

following up on #34 is there still a plan to rewrite the repo to typescript or include typings?

**Comment by mroloux** [comment]
Hi!

Rewriting this repo to TS (or including typings) is not planned at the moment. I agree that it would be awesome, but we have other priorities at the moment.

Hope that makes sense.

**Comment by BengtHagemeister** [comment]
Thanks for the very quick answer! 

Is there maybe an OpenAPI / Swagger specification for the Seatsio API? Then I could auto generate the typescript client with openapi-generator (https://openapi-generator.tech/).

**Comment by mroloux** [comment]
No, there's no OpenAPI spec for our API. But then again, it's a good suggestion, and something we're considering - just not in the short term.

**Comment by BengtHagemeister** [comment]
Ok thanks! 

A general OpenAPI spec where all clients are derived from could in the long term simplify the development of SDK significantly. The people from ORY are doing that quite well: https://github.com/ory/sdk

**Comment by J-Gonzalez** [comment]
This is something we're looking for as well. We've got a new backend platform, and lack of typescript support for this paid product is disheartening. 

There is a @types definition for the renderer JS client (though not the react client), but nothing for the API client. I agree that an OpenAPI spec (or just default TS typings) would be super helpful.

**Comment by mroloux** [comment]
Hi, a TS client is on our roadmap. Will let you know when there's more news about that.

**Comment by mroloux** [comment]
We support typescript since [v76.0.0](https://github.com/seatsio/seatsio-js/releases/tag/v76.0.0)
